lib/src/mock_query.dart

```diff
@@ -122,6 +128,37 @@ class MockQuery extends Mock implements Query {
     });
   }
 
+  @override
+  Query startAt(List<dynamic> values) =>
+      _cursorUtil(values, (docs, index) => docs.sublist(max(0, index)));
+
+  @override
+  Query endAt(List<dynamic> values) => _cursorUtil(values,
+      (docs, index) => docs.sublist(0, index == -1 ? docs.length : index));
+
+  Query _cursorUtil(
```

Comment:
It'd help make the code slightly more readable if you passed `List<String> orderByKeys` as a parameter to this function. Would you mind doing that, so that _cursorUtil doesn't access parameters directly?